src/processes/follow-topic-process.js

```diff
@@ -1,4 +1,4 @@
-const StreamChat = require('stream-chat').StreamChat;
+const {StreamChat} = require('stream-chat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `const StreamChat = require('stream-chat').StreamChat;` to `const {StreamChat} = require('stream-chat');` is a good one. It uses destructuring assignment, which is a more modern and cleaner way to import modules in JavaScript. This change should not affect the functionality of the code.
```diff
- const StreamChat = require('stream-chat').StreamChat;
+ const {StreamChat} = require('stream-chat');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->